### PR TITLE
BP-764: Update blueprint to Angular 16

### PR DESCRIPTION
**Pull request by msljivic**

### CODE REVIEW

File: enigmatry-entry-blueprint-app/src/app/core/auth/auth.guard.spec.ts [hunk 1]
```diff
@@ -42,8 +42,8 @@ describe.each(userCases)('Testing authentication guard...', userCase => {
             const result = await act();
 
             expect(result).toBe(userCase.allowedAccess);
-            expect(loginRedirectMock).toBeCalledTimes(userCase.redirectedToLogin ? 1 : 0);
```
Comment (msljivic) [inline enigmatry-entry-blueprint-app/src/app/core/auth/auth.guard.spec.ts:45]: 'toBeCalledTimes' is deprecated in favor of 'toHaveBeenCalledTimes'

File: enigmatry-entry-blueprint-app/src/app/core/auth/auth.interceptor.spec.ts [hunk 2]
```diff
@@ -42,37 +42,37 @@ beforeEach(async() => {
 describe('Testing auth interceptor...', () => {
   it(`should do nothing if it's non-API request`, (done: jest.DoneCallback) => {
     tester.requestSuccess(done, () => {
-      expect(tokenMock).not.toBeCalled();
```
Comment (msljivic) [inline enigmatry-entry-blueprint-app/src/app/core/auth/auth.interceptor.spec.ts:45]: 'toBeCalled' is deprecated in favor of 'toHaveBeenCalled'

File: enigmatry-entry-blueprint-app/src/app/core/auth/auth.interceptor.ts [hunk 3]
```diff
@@ -25,6 +25,7 @@ export class AuthInterceptor implements HttpInterceptor {
       .pipe(
         switchMap(accessToken => next.handle(this.addAuthorizationHeader(req, accessToken))),
         catchError(async error => {
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-enum-comparison
           if (error instanceof HttpErrorResponse && error.status === HttpStatusCode.Unauthorized) {
```
Comment (msljivic) [inline enigmatry-entry-blueprint-app/src/app/core/auth/auth.interceptor.ts:29]: we are comparing status (number) === HttpStatusCode.Unauthorized (number)
this is not according to 'no-unsafe-enum-comparison'

**Review by zsrdjan — APPROVED** [review]

File: enigmatry-entry-blueprint-app/src/styles/partials/core/layouts/_visibility.scss [hunk 4]
```diff
@@ -8,6 +8,7 @@
 	}
 }
 
+/* stylelint-disable-next-line plugin/stylelint-group-selectors */
```
Comment (msljivic) [inline enigmatry-entry-blueprint-app/src/styles/partials/core/layouts/_visibility.scss:11]: " .hide-on-tablet and .hide-on-mobile have same properties, group them.  plugin/stylelint-group-selectors"

disabled rule for this line, .hide-on-table and .hide-on-mobile don't look the same to me

**Pull request merged**